client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -675,6 +714,8 @@ export class MotionDetailComponent extends BaseViewComponent implements OnInit {
         if (this.newMotion) {
             this.createMotion();
         } else {
+            // When saving the changes, notify other users if they edit the same motion.
+            this.unsubscribeEditNotifications(TypeOfNotificationViewMotion.TYPE_SAVING_EDITING_MOTION);
```

Comment:
Please change the order: The update must be successfull to remove to publish this message.

Reply:
calling `updateMotionFromForm` should be sufficient I suppose